code/modules/mob/living/carbon/human/life.dm

```diff
@@ -264,6 +264,16 @@ var/global/list/brutefireloss_overlays = list("1" = image("icon" = 'icons/mob/sc
 				gene.OnMobLife(src)
 
 		if (radiation)
+
+			if((locate(src.internal_organs_by_name["resonant crystal"]) in src.internal_organs))

```

Comment:
i was thinking more along the lines of:
for(var/obj/item/organ/dongs in internal_organs_list)
>dongs.handle_effects()